Dockerfile

```diff
@@ -18,6 +18,10 @@ RUN apt-get --no-install-recommends -y install \
     libmozjs185-dev \
     python
 
+# Install Nodejs
+RUN curl -sL https://deb.nodesource.com/setup | bash -	
```

Comment:
+1, we should do what the script does directly, not download it first. It is MIT licensed, we could keep a local copy.